.vscode/cspell.json

```diff
@@ -163,4 +173,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
```suggestion
}

```

Reply:
THe formatter doesn't add that empty line.